theme/open/views/dataprivacy/privacy_anschrift.hbs

```diff
@@ -0,0 +1,6 @@
+<p>
```

Comment:
we have seen this information before, reuse it :)

Reply:
It is different, too